### ISSUE TITLE
[Urgent]RuntimeError: Device with "FPGA" name is not registered in the InferenceEngine

### ISSUE THREAD

**Issue by NSVR57** [openvinotoolkit/openvino#3000]
Hi I am currenly using linux machine with intel Arria 10 card. while doing Hetro inference i am getting the following error.
```

File "Image_classification.py", line 92, in main
    exec_net = ie.load_network(network=net, device_name=args.device)
  File "ie_api.pyx", line 311, in openvino.inference_engine.ie_api.IECore.load_network
  File "ie_api.pyx", line 320, in openvino.inference_engine.ie_api.IECore.load_network
RuntimeError: Device with "FPGA" name is not registered in the InferenceEngine
```

**Comment by brmarkus** [comment]
Do you use OpenVINO from source-code and built it on your own, or do you use the pre-built installation from `software.intel.com`? I think the open-source version of OpenVINO doesn't contain the FPGA plugin.
DO you have OpenVINO installed as described under `https://docs.openvinotoolkit.org/2020.3/_docs_install_guides_installing_openvino_linux_fpga.html`?

**Comment by NSVR57** [comment]
Hi @brmarkus 

I have installed from the same link that you have provided

**Comment by brmarkus** [comment]
Which exact FPGA model do you have - and have installed which exact OpenVINO version?
On the installation description page I mentioned above it says:

> NOTE: Intel® Arria 10 FPGA (Mustang-F100-A10) SG1 is no longer supported. If you use Intel® Vision Accelerator Design with an Intel® Arria 10 FPGA (Mustang-F100-A10) Speed Grade 1, we recommend continuing to use the Intel® Distribution of OpenVINO™ toolkit 2020.1 release.

And

> NOTE: Intel® Arria® 10 GX FPGA Development Kit is no longer supported. For the Intel® Arria® 10 FPGA GX Development Kit configuration guide, refer to the 2019 R1.1 documentation.

From the installation description do you see the verification samples working as under "Run the Verification Scripts to Verify Installation and Compile Samples" and "Hello World Face Detection Tutorial"?

Please check the section "Install and Configure Your Compatible Hardware" as well.

**Comment by brmarkus** [comment]
Have you tried the C/C++/Python version of the sample `hello_query_device`? Does it list `FPGA` as a supported/detected/probed device-plugin?

**Comment by ehong-w** [comment]
Hi, I am facing the same issue on DevCloud EDGE. I ran the `hello_query_device.py`, `FPGA` was not shown under it.
![image](https://user-images.githubusercontent.com/68590570/116789260-9be7fb80-aae0-11eb-9126-44c64ce8bb3c.png)
![image](https://user-images.githubusercontent.com/68590570/116789273-a4403680-aae0-11eb-99f1-ca479a7fddd0.png)
![image](https://user-images.githubusercontent.com/68590570/116789277-ac987180-aae0-11eb-96aa-1030c73c9be0.png)

This is the error when I ran some inference specifying `-d FPGA` or `-d HETERO:FPGA,CPU`. I am on OpenVINO 2021.3, I did realize that FPGA plugin is no longer supported, but it's still advertised that EDGE can perform FPGA inference, am I still able to perform inference using the FPGA plugin?
![image](https://user-images.githubusercontent.com/68590570/116789251-8d99df80-aae0-11eb-8952-27eaefef01b5.png)

If no, can anyone provide guidance on using FPGA on DevCloud EDGE?
**The FPGA I mentioned is Arria 10 GX PAC.**

Thank you!
